packages/server/src/routes/members/give.ts

```diff
@@ -1,23 +1,38 @@
 import { CollectionReference, Firestore } from "@google-cloud/firestore";
-import { firestore, messaging as adminMessaging } from "firebase-admin";
+import { firestore, messaging as adminMessaging, auth } from "firebase-admin";
 import Big from "big.js";
 
 import {
   GiveOperation,
   Operation,
-  OperationType
+  OperationType,
+  GiveContent,
+  GiveContentType
 } from "@raha/api-shared/dist/models/Operation";
 import { InsufficientBalanceError } from "@raha/api-shared/dist/errors/RahaApiError/members/give/InsufficientBalanceError";
-import { GiveApiEndpoint } from "@raha/api-shared/dist/routes/members/definitions";
 import { NotFoundError } from "@raha/api-shared/dist/errors/RahaApiError/NotFoundError";
 
 import { createApiRoute, OperationToInsert } from "..";
 import { getMemberById } from "../../collections/members";
 import { sendPushNotification } from "../../helpers/sendPushNotification";
+import { GiveApiEndpoint } from "@raha/api-shared/dist/routes/members/definitions";
 
 const DEFAULT_DONATION_RECIPIENT_UID = "RAHA";
 const DEFAULT_DONATION_RATE = 0.03;
 
+function _notificationMessageForGive(
+  senderName: string,
+  amount: string,
+  memo?: string,
+  content?: GiveContent
+) {
+  const prefix = `${senderName} gave you ${amount} Raha`;
+  const suffixes = [
+    memo ? `for "${memo}"` : undefined,
+    content ? "and attached media! 🖼" : undefined
```

Comment:
Hmm I think I would omit this or use a different word than "media". "attached a photo" or "attached a video" might be more meaningful